ChangeLog.md

```diff
@@ -31,6 +31,17 @@ See docs/process.md for more on how version tagging works.
   overflow will trap rather corrupting global data first).  This should not
   be a user-visible change (unless your program does something very odd such
   depending on the specific location of stack data in memory). (#18154)
+- Default value for `STACK_SIZE` (the size of the wasm shadow stack) was reduced
+  from 5Mb to 64Mb.  Project that use more that 64Kb of stack will now have to
```

Comment:
Typo: 64Mb -> 64Kb.

Also lower case b is generally read as bit, and upper case B is generally read as a byte, so I'd recommend using 5MB and 64KB instead of Mb and Kb?